operator/duties/scheduler.go

```diff
@@ -151,6 +152,7 @@ func NewScheduler(opts *SchedulerOptions) *Scheduler {
 			NewSyncCommitteeHandler(dutyStore.SyncCommittee),
 			NewVoluntaryExitHandler(opts.ValidatorExitCh),
 			NewCommitteeHandler(dutyStore.Attester, dutyStore.SyncCommittee),
+			NewForkListener(opts.P2PNetwork),
```

Comment:
moved `ForkListener` to scheduler, not sure that its very elegant, but less changes anyway....